### PR TITLE
Update helm charts

### PR DESCRIPTION
**Pull request by MujkicA**
Closses #95

### CODE REVIEW

File: helm/fuel-block-committer/templates/deployment.yaml [hunk 1]
```diff
@@ -62,12 +57,18 @@ spec:
           volumeMounts:
             - name: {{ include "fuel-block-committer.fullname" . }}-vol
               mountPath: "/mnt/db"
```
Comment (segfault-magnet) [inline helm/fuel-block-committer/templates/deployment.yaml:59]: Do we still need this mount point since we're now connecting to a remote db?

**Review by Br1ght0ne — APPROVED** [review]

**Review by Salka1988 — APPROVED** [review]

**Comment by rfuelsh** [comment]
@MujkicA - thanks for the PR

1) this file is empty- helm/fuel-block-committer/templates/config.yaml

2) how does the config.yaml get places in that mount path -  "/mnt/config/config.yaml"- it needs to be there within the container - like get copied in Dockerfile

**Comment by rfuelsh** [comment]
@MujkicA - sent some comments over slack but refer to https://dev.to/joshduffney/kubernetes-using-configmap-subpaths-to-mount-files-3a1i- your config.yaml is empty will need to see that to assess how this works- but the article I will sent will show you to use a subpath- everything looks good overall- just some changes

**Comment by rfuelsh** [comment]
hi @MujkicA -

- your configmap has one extra data field in it - I don't think line 6 should be there

- For line 71 - given you reference "committer-config" key- in helm/fuel-block-committer/templates/deployment.yaml it should be your configmap otherwise I think deployment will fail

**Review by rfuelsh — APPROVED** [review]

**Pull request merged**